docs/faq.rst

```diff
@@ -557,6 +557,8 @@ These are your options to protect against that:
 - Mount C's filesystem on another machine and then create a backup of it.
 - Do not give C filesystem-level access to S.
 
+See :ref:`hosting_secure_repositories` for a detailed protection guide.
```

Comment:
just "detaillled guide"? (also note the 2 "L")

too much "secure" and "protection" and it somehow reads like a virus scanner advertisement. :-)